### PR TITLE
coerce URI to use HTTPS protocol

### LINKED ISSUE #1826: Make the Fax Service link less finicky
* **What are we trying to do?**
We have the fax URL set as a user configurable option. Right now if the user forgets to type `https://` at the front of the link in the Configuration Management entry, the link at the bottom of the page is broken. It works fine if they have the https, but let's make the app worry about this instead of putting it on our users! Let's make it work regardless of https or not.

#### Broken
Screenshot from 2019-10-18 21-00-56

#### Working!
Screenshot from 2019-10-18 21-01-27

### PR DESCRIPTION
**Pull request by Kevin-Wei**
I rule and have completed some work on Case Manager that's ready for review!

Use new URIService Utility 

This pull request makes the following changes:

(If there are changes to the views, please include a screenshot so we know what to look for!)

It relates to the following issue #s: 
* Fixes #1826

### CODE REVIEW

**Comment by lomky** [comment]
Closing in favor of #1845

**Comment by Kevin-Wei** [comment]
Will be refactoring to use the utility in #1845

File: app/services/uri_service.rb [hunk 1]
```diff
@@ -2,14 +2,17 @@ class URIService
   attr_accessor :uri
 
   def initialize(uri)
+    return nil if uri.nil?
```
Comment (Kevin-Wei) [inline app/services/uri_service.rb]: Not a huge fan of this PORO handling nil initializations

File: app/services/uri_service.rb [hunk 2]
```diff
@@ -2,14 +2,17 @@ class URIService
...
       @uri = URI.parse(URI.encode(uri))
     rescue URI::InvalidURIError
       nil
     end
   end
 
-  def secure_scheme!
+  def secure_scheme_uri!
     @uri.scheme = "https"
+    @uri
```
Comment (Kevin-Wei) [inline app/services/uri_service.rb]: added this return here to and changed method name to make the interface to the PORO friendlier

**Comment by colinxfleming** [comment]
looks good to me but want to get @lomky 's review as the issue reporter here.

**Comment by Kevin-Wei** [comment]
if we need something more robust, its possible [this gem](https://github.com/sporkmonger/addressable) handles the edgiest of edge cases.

**Comment by colinxfleming** [comment]
@lomky ready for another pass

**Pull request merged**